mobs/owl.lua

```diff
@@ -1,46 +1,84 @@
 mobs:register_mob("dmobs:owl", {
-	type = "animal",
-	passive = true,
-	hp_min = 12,
-	hp_max = 22,
-	armor = 130,
-	collisionbox = {-0.3, -0.5, -0.3, 0.2, 0.2, 0.2},
-	visual = "mesh",
-	mesh = "owl.b3d",
-	textures = {
-		{"dmobs_owl.png"},
+    type = "animal",
+    passive = true,
+    hp_min = 12,
+    hp_max = 22,
+    armor = 130,
+    collisionbox = {-0.3, -0.5, -0.3, 0.2, 0.2, 0.2},
+    visual = "mesh",
+    mesh = "owl.b3d",
+    textures = {
+        {"dmobs_owl.png"},
+    },
+    mobs:register_mob("dmobs:owl", {
+    blood_texture = "mobs_blood.png",
+    visual_size = {x=2, y=2},
+    makes_footstep_sound = false,
+    walk_velocity = 0,
+    run_velocity = 0,
+    jump = false,
+    water_damage = 2,
+    lava_damage = 2,
+    light_damage = 0,
+    view_range = 1,
+   	follow = {
+		"farming:seed_wheat",
+		"farming:seed_cotton",
+		"mobs_bugslive:bug",
+		"mobs_butterfly:butterfly",
+		"butterflies:butterfly_red",
+		"butterflies:butterfly_violet",
+		"butterflies:butterfly_white",
+		"fireflies:firefly",
+                "mobs_animals:rat",
```

Comment:
⚠️ Extra indents hint at addded code blocks - visual complexity during reading. Not a breaker, but not ideal ...